sql/trace.go

```diff
@@ -79,20 +80,35 @@ func (n *explainTraceNode) Next() bool {
 			basePos = n.lastPos + 1
 		}
 
+		// Iterate through once to determine earliest timestamp.
+		var earliest time.Time
+		for _, sp := range n.txn.CollectedSpans {
+			for _, entry := range sp.Logs {
+				if entry.Timestamp.Before(earliest) || earliest.IsZero() {
+					earliest = entry.Timestamp
+					break
+				}
+			}
+		}
+
 		for _, sp := range n.txn.CollectedSpans {
 			for i, entry := range sp.Logs {
 				var timeVal string
 				if i > 0 {
-					timeVal = time.Duration(entry.Timestamp.Sub(n.lastTS)).String()
+					timeVal = fmt.Sprintf("%.3fms", time.Duration(entry.Timestamp.Sub(n.lastTS)).Seconds()*1000)
```

Comment:
while you're here, can you call this `duration` and the below `commulativeDuration` for consistency?